Steamworks/src/org/usfirst/frc/team2363/robot/commands/PixyCam/PixyCommand.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team2363.robot.commands.PixyCam;
+
+import static org.usfirst.frc.team2363.robot.Robot.*;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class PixyCommand extends Command {
+	
+	int Signature = 1;
+	
+    public PixyCommand() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	// needs code from the gearGrabber subsystem
+        requires(pixy);
+      
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	if (pixy.isOff()) {
```

Comment:
Why yes it is...  will have to fix that... My focus was on just getting communication with the pixy.  I was not as worried about how it would be called, since I know that will change.